core/src/main/java/ai/timefold/solver/core/impl/constructionheuristic/DefaultConstructionHeuristicPhaseFactory.java

```diff
@@ -79,7 +80,7 @@ protected DefaultConstructionHeuristicPhaseBuilder<Solution_> createBuilder(
     @Override
     public ConstructionHeuristicPhase<Solution_> buildPhase(int phaseIndex, boolean lastInitializingPhase,
             HeuristicConfigPolicy<Solution_> solverConfigPolicy, BestSolutionRecaller<Solution_> bestSolutionRecaller,
-            SolverTermination<Solution_> solverTermination) {
+            SolverTermination<Solution_> solverTermination, ValueRangeManager<Solution_> valueRangeManager) {
```

Comment:
We have a significant number of parameters now. Should we consider adding a Builder?

Reply:
Maybe at some point. I don't want to make this PR bigger than it already is, with unrelated changes.